lib/src/chart/line_chart/line_chart_data.dart

```diff
@@ -198,4 +204,37 @@ class FlDotData {
     this.dotSize = 4.0,
     this.checkToShowDot = showAllDots,
   });
-}
\ No newline at end of file
+}
+
+class VerticalLine extends FlLine {
+  final double x;
+
+  const VerticalLine({
+    @required this.x,
+    Color color = Colors.black,
+    double strokeWidth = 1,
+  }) : super(color: color, strokeWidth: strokeWidth);
+}
+
+class HorizontalLine extends FlLine {
+  final double y;
```

Comment:
We have a convention on Library, 
the horizontal is all of the things that draw on the x axis,
the vertical is all of the things that draw on the y axis,
then HorizontalLine means lines that draw on the x axis and ( for example in x = 1, 2, 3, 4, 5), and y is the same  (may be it is VerticalLine in your head, but it is a convention in this library, check the FlTitlesData you can find out what does it means.)
conclusion:
we should have x here instead of y

Reply:
Won't this confuse everybody? A horizontal line has infinite x values... seems strange but I'll change, its your library.

Reply:
Y has been changed to x.